dev/cnf/gradle/scripts/fat.gradle

```diff
@@ -324,6 +331,69 @@ assemble {
   }
 }
 
+ext.gitClone = { proj, branch = 'NOT_SET', folder = '', site = 'git@github.com', org = 'eclipse' ->
```

Comment:
I don't think this mechanism will be widely enough used that it should belong in the common fat.gradle script.  We should move this to the specific FAT that is requiring this functionality, and if it gets more widely used (i.e. >3 places) we can perhaps move it to a dedicated gradle file, such as "fat_tck.gradle" or something.

Reply:
We expect there to be one of these projects per 'function' of microprofile project.  (Currently there are 22 TCK's in my dev folder but this is one for each version of, for example Config etc.)
I am not that opinionated if it ends up somewhere else :-)

Reply:
ok, at the very least I think it's safe to say this is a sufficiently different "style" of FATs that would warrant its own gradle file, which could certainly extend from the existing fat.gradle file